ds/beis_indicators/nomis/ni_processing.py

```diff
@@ -0,0 +1,77 @@
+import pandas as pd
+import beis_indicators
+
+PROJECT_DIR = beis_indicators.project_dir
+NI_BRES_2019_PUB_TABLES = "https://www.nisra.gov.uk/sites/nisra.gov.uk/files/publications/BRES2019-Publication-Tables.xlsx"
+
+
+def sic4_to_cluster_lookup(
+    sic4_lookup_csv=f"{PROJECT_DIR}/data/raw/sic_4_industry_segment_lookup.csv",
+):
+    """Use SIC4 to cluster namelookup csv file to create a dictionary
+    with SIC4 as keys, cluster_name as values"""
+    df = pd.read_csv(sic4_lookup_csv)
+    lookup = df[["sic_4", "cluster"]].rename(
+        columns={"sic_4": "SIC4", "cluster": "cluster_name"}
+    )
+    return dict(zip(lookup.SIC4, lookup.cluster_name))
```

Comment:
```suggestion
    lookup = pd.read_csv(sic4_lookup_csv)
    return dict(zip(lookup.sic_4, lookup.cluster))
```

Reply:
I thought the lookup had to have the same column names as the dataframe that it is being mapped to. Good to know that it does not, thanks.